### DISCUSSION TITLE
Can I change some variables in docker-compose file after the container is already created?

### DISCUSSION THREAD

**Discussion by apoorv569** [nextcloud/all-in-one#1550] [Questions] [answered]
Can I increase `NEXTCLOUD_MAX_TIME`, `NEXTCLOUD_UPLOAD_LIMIT` and `NEXTCLOUD_MEMORY_LIMIT` after I have already created the container and has been used for 3-4 days already?

**Comment by Zoey2936** [comment, marked as answer]
Yes, just stop and delete the mastercontainer and then recreate it with the new environment variables
```
docker stop nextcloud-aio-mastercontainer
docker rm nextcloud-aio-mastercontainer
docker run [your docker run command]
```

**Reply by apoorv569** [reply]
> Yes, just stop and delete the mastercontainer and then recreate it with the new environment variables
> 
> ```
> docker stop nextcloud-aio-mastercontainer
> docker rm nextcloud-aio-mastercontainer
> docker run [your docker run command]
> ```

Oh! Nice. And I won't loose any data by deleting and re-creating `mastercontainer`  like users and their files, right?

**Reply by Zoey2936** [reply]
no

**Reply by Zoey2936** [reply]
These are saved in volumes, and you are only deleting the container, not the volumes
